python/src/aiconfig/editor/README.md

```diff
@@ -1,39 +1,42 @@
 # Editor
 
-# Install:
+## Usage (prod)
+
+### Install:
 
 Install python-aiconfig from pip, then set `aiconfig` alias (in shell, .bashrc, .zshrc, etc.)
 
 One-liner:
 `pip install python-aiconfig; alias aiconfig="python -m 'aiconfig.scripts.aiconfig_cli'"`
 
-## Usage
+
+### Run
 
 `aiconfig edit  --aiconfig-path=/my/path'`
 
 ```
 [INFO] 2023-12-18 23:54:14,379 server.py:32: Edit config: {
-  "server_port": 8888,
+  "server_port": 8080,
   "aiconfig_path": "/my/path"
 }
-[INFO] 2023-12-18 23:54:14,379 server.py:33: Editor server running on http://localhost:8888
+[INFO] 2023-12-18 23:54:14,379 server.py:33: Editor server running on http://localhost:8080
 ```
 
 Go to url in browser to use app.
 
 ## Dev
 
-1. Edit
-   Server code: `python/src/editor/server/server.py`
-   Client code: `aiconfig/python/src/editor/client/src`
+### Install:
 
-2. Start react app:
 
-```
-cd python/src/editor/client && yarn && yarn start
-```
+`pip install -e path/to/local/aiconfig/python`
+`alias aiconfig="python -m 'aiconfig.scripts.aiconfig_cli'"``
+
+2. Run backend and frontend servers:
+
+`aiconfig edit  --aiconfig-path=/my/path --server-port=8080 --server-mode=debug`
```

Comment:
By default server port is just 8080 right? Would it be simpler to do without it?